### PR TITLE
[ACS-5400] Fix incomplete string escaping 

### PR DESCRIPTION
**Pull request by nikita-web-ua**
**Please check if the PR fulfills these requirements**

> - [X] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [X] Bugfix
> - [ ] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

https://alfresco.atlassian.net/browse/ACS-5400

**What is the new behaviour?**

Fixed incomplete string escaping

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Review by AleksanderSklorz — APPROVED** [review]

File: lib/cli/scripts/changelog.ts [hunk 1]
```diff
@@ -109,7 +109,7 @@ function getCommits(options: DiffOptions): Array<Commit> {
     let log = shell.exec(command, { cwd: options.dir, silent: true }).toString();
 
     // https://stackoverflow.com/a/13928240/14644447
-    log = log.trim().replace(/"/gm, '\\"').replace(/\^@\^/gm, '"');
+    log = JSON.stringify(log.trim()).slice(1, -1).replace(/\^@\^/gm, '"');
```
Comment (DenysVuika) [inline lib/cli/scripts/changelog.ts:112]: i'm not sure it is going to work as log is not an object so you can't stringify it

**Review by DenysVuika — CHANGES_REQUESTED** [review]

File: lib/cli/scripts/changelog.ts (hunk 1 above)
Comment (nikita-web-ua) [reply lib/cli/scripts/changelog.ts:112]: `log` is a string, we use `JSON.stringity` to escape the quotes

Comment (DenysVuika) [reply lib/cli/scripts/changelog.ts:112]: did you test the changelog command? does it produce the expected output?

Comment (nikita-web-ua) [reply lib/cli/scripts/changelog.ts:112]: Added a fix, now works as expected

**Review by MichalKinas — APPROVED** [review]

**Review by DenysVuika — APPROVED** [review]

**Pull request merged**